org.civicrm.bootstrapcivicrm/scss/civicrm/_mixins.scss

```diff
@@ -15,4 +15,18 @@
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box;
+}
+
+// A transformation is an effect that lets an element change shape, size and position.
```

Comment:
Maybe i did't just explain it enough. Can you have a look on 
https://github.com/civicrm/civihr/blob/70663b29cb7c5183cffe420adfd29af13fff983e/org.civicrm.bootstrapcivicrm/scss/civicrm/_mixins.scss

each mixin has a different annotation format 

```
/*
 * It use for calculate height and width the element
 */

// =======================
```

```
/* mixin for box sizing */
```

```
// A transformation is an effect that lets an element change shape, size and position.
```

Can you standarize the annotation format across this file?